contrib/hack/gen-deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
I think we could just create a folder `scripts` and put this and the other files into it?  Or a folder called `hack` in the same way kubernetes does.  Doesn't matter to me, if you have a preference.  Also, open to `contrib` if you have a particular reason there.  

Do you have any suggestions on how this should be integrated into CI pipeline?  I see that it generates the file, should it be before/during?  I am thinking of adding it as a step to the version step in the makefile.  

Another note, the `values/adapter.yaml` file is empty.  Should that contain the override values from the default chart values files?  Currently I don't think the there are any values we want to override it with right now. So either add a comment to the values file that states it can contain over rides or remove it.

Reply:
I'll move this down to `hack` instead of `contrib/hack`, since this code will be maintained by core developers.

For where this should occur, I'd favor during/after the version step. Given that during this phase the new version would be known for the Helm chart to utilize that could be of favor.

Reply:
sounds good thanks!